core/runtime/binaryen/runtime_manager.hpp

```diff
@@ -0,0 +1,62 @@
+/**
+ * Copyright Soramitsu Co., Ltd. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef KAGOME_CORE_RUNTIME_BINARYEN_RUNTIME_API_RUNTIME_MANAGER
+#define KAGOME_CORE_RUNTIME_BINARYEN_RUNTIME_API_RUNTIME_MANAGER
+
+#include <binaryen/wasm-interpreter.h>
+
+#include "common/blob.hpp"
+#include "common/logger.hpp"
+#include "crypto/hasher.hpp"
+#include "extensions/extension_factory.hpp"
+#include "outcome/outcome.hpp"
+#include "runtime/wasm_provider.hpp"
+#include "runtime_external_interface.hpp"
+
+namespace kagome::runtime::binaryen {
+
+  class RuntimeManager {
```

Comment:
Please comment.
Also RuntimeManager name seems to generic, but I can't think of a name which would be less generic.